src/app/CASESessionManager.h

```diff
@@ -71,9 +78,9 @@ class CASESessionManager : public OperationalSessionReleaseDelegate
     void FindOrEstablishSession(const ScopedNodeId & peerId, Callback::Callback<OnDeviceConnected> * onConnection,
                                 Callback::Callback<OnDeviceConnectionFailure> * onFailure);
 
-    OperationalSessionSetup * FindExistingSessionSetup(const ScopedNodeId & peerId) const;
+    OperationalSessionSetup * FindExistingSessionSetup(const ScopedNodeId & peerId, bool isSessionDesignatedLookup) const;
```

Comment:
I would call that second arg `forAddressUpdate` throughout...  And probably default it to false here, so we don't need the device controller changes.